brush-interactive/src/completion.rs

```diff
@@ -72,6 +100,21 @@ fn postprocess_completion_candidate(
                 candidate.push(std::path::MAIN_SEPARATOR);
             }
         }
+
+        if !options.no_autoquote_filenames {
+            let quote_mode = match quote_char {
+                Some(q) => {
```

Comment:
Minor style request: would you please flatten the `match` and the `if`/`else` underneath it. Something like the following?

```rust
let quote_mode = match quote_char {
    Some('\'') => escape::QuoteMode::SingleQuote,
    Some('\"') => escape::QuoteMode::DoubleQuote,
    None | _ => escape::QuoteMode::BackslashEscape,
};
```